src/main/java/net/greatstart/config/GreatStartWebAppInitializer.java

```diff
@@ -0,0 +1,23 @@
+//package ua.petshop.config;
+//
+//import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
+//
+//public class PetShopWebAppInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
+//
+//    @Override
+//    protected String[] getServletMappings() {
+//        return new String[] {"/"};
+//    }
+//
+//    @Override
+//    protected Class<?>[] getRootConfigClasses() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)